cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-configmaps-secrets-env-var.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-configmaps-secrets-envvar"]
+= Tutorial: ConfigMaps, secrets, and environment variables
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-configmaps-secrets-envvar
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 05-07-2024
+
+This tutorial shows how to configure the OSToy application by using xref:../../nodes/pods/nodes-pods-configmaps.adoc[config maps], link:https://docs.openshift.com/container-platform/latest/cicd/builds/creating-build-inputs.html#builds-input-secrets-configmaps_creating-build-inputs[secrets], and link:https://docs.openshift.com/container-platform/3.11/dev_guide/environment_variables.html[environment variables]. See these linked topics for more information.
+
+== Configuration using ConfigMaps
+Config maps allow you to decouple configuration artifacts from container image content to keep containerized applications portable.
+
+.Procedure
+* In the OSToy app, click *Config Maps* in the left menu, displaying the contents of the config map available to the OSToy application. The code snippet shows an example of a config map configuration:
```

Comment:
Provide where to click before what to click. So rewrite as: "..., in the left menu, click..."